shared/util/simple-validators.js

```diff
@@ -25,37 +25,41 @@ function isEmptyOrBlank(thing: ?string): boolean {
 }
 
 // Returns an error string if not valid
-function isValidCommon(thing: ?string): ?Error {
-  if (isEmptyOrBlank(thing)) return new Error('Cannot be blank')
```

Comment:
lets not return errors from functions like this. all callers just pull the error string out